src/realm/sync/subscriptions.cpp

```diff
@@ -440,7 +448,7 @@ SubscriptionStore::SubscriptionStore(DBRef db)
         auto version_obj = schema_metadata->get_object(0);
         auto version = version_obj.get<int64_t>(
             lookup_and_validate_column(schema_metadata, c_flx_meta_schema_version_field, type_Int));
-        if (version != 1) {
+        if (version != 2) {
```

Comment:
since this isn't used anywhere it doesn't matter at this point, but you could start an upgrade pattern like this:
```
if (version == 1) {
// add the version 2 column
}
// set version to 2
```